### DISCUSSION TITLE
Introduction to WebAssembly and Rust

### DISCUSSION THREAD

**Discussion by giscus** [BojanStipic/website#305] [Post Comments]
# Introduction to WebAssembly and Rust

https://bojanstipic.rs/blog/01-introduction-to-webassembly-and-rust/

**Comment by BojanStipic** [comment]
Comments section powered by [giscus](https://giscus.app/) is now available!

**Comment by aag1992** [comment]
Super interesting to read. I'm looking into constructing a lecture to some colleagues. Do you have any good examples that show powerful usages of WASI?

**Reply by BojanStipic** [reply]
I haven't played much with WASI just yet, but I have recently started using [Zellij](https://zellij.dev/) terminal multiplexer (similar to tmux) which seems to have a WASM+WASI based plugin system, so you might want to take a look at that.
